apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md

```diff
@@ -0,0 +1,22 @@
+Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user feel at home on macOS and keep the experience quality high.
+
+Buttons are a standard control that users can interact with to take actions within your application. The FluentUI button for macOS adds system appropriate hover effects when your mouse rests over it.
+
+<DisplayToggle onText="Dark" offText="Light" label="Theme Switcher">
+
+### Primary Filled
+
+<img className="off" src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200701.001/fabric-website/images/controls/macos/Button/button_primaryfilled_light.png?text=LightMode" />
```

Comment:
Guessing this is really feedback for @micahgodbolt but we should be careful about using such generic class names as `on` and `off` due to conflict potential.